docs/data-sources/anycast_configs.md

```diff
@@ -10,7 +10,25 @@ description: |-
 
 Retrieve all named anycast configurations for the account.
 
-
+## Example Usage
+
+```terraform
+#Get anycast configuration by tag filter
+data "bloxone_anycast_ac_configs" "example_filter" {
+  tag_filters = {
+    tag1 = "value1"
+  }
+}
+
+#Get anycast configuration by service
+data "bloxone_anycast_ac_configs" "example_service" {
```

Comment:
name hasn't been renamed in the example.

Reply:
done